src/lint/utils.ts

```diff
@@ -73,8 +73,8 @@ export function rhsMatches(a: RightHandSide | OneOfList, b: RightHandSide | OneO
       }
       return symbolSpanMatches(aHead, bHead);
     }
-    default:
-      throw new Error('unknown rhs type ' + a.constructor.name);
+    case SyntaxKind.OneOfList:
+      return oneOfListMatches(a, b as OneOfList);
```

Comment:
This needs a `default`. (If TS complains, add a `ts-expect-error`.)

I can do this myself when landing if you don't get to it first.

Reply:
Done